### ISSUE TITLE
Feature request: parallel download from multiple accounts

### ISSUE THREAD

**Issue by vasyugan** [nextcloud/desktop#1508]
With multiple accounts, when you have one account with a rather slow connection, it would make particularly good sense to have the client download from the other accounts in parallel rather than having them wait until this one account is done. I am this seeing right now, one account serves data at less than 100 kbit/sec and the others would be potentially much faster, but at this speed I will have to wait for hours until they even start syncing.

**Comment by CryptoSiD** [comment]
Any hope we'll get this feature?

**Comment by joshtrichards** [comment]
Related: #2376 #2452

**Comment by AdamAnon** [comment]
Still an issue in 2024. I have a slow connection to hosted instance that will take 10+ days for the initial sync, and during this time my local instance will not be synchronizing? This is absurd! Even the bloatware that Dropbox is can sync multiple accounts simultaneously!

**Comment by Rello** [comment]
Hello,

this ticket is pretty old and the client had several fundamental redesigns.
For your usecase, VFS might be a better option as it does not download everything.
If the issue still exists, please create a new ticket with current information.

Thank you

**Comment by q-wertz** [comment]
Sorry to disagree that this is a solution for the problem described above.

AFAIK: 
- there is no stable VFS for Linux (e.g. https://github.com/nextcloud/desktop/issues/3668)
- even VFS needs to sync to the server, just the download is smaller. In my understanding a slow server (in regards of response time and so on) still will take a lot of time to initially sync using VFS

**Comment by Rello** [comment]
Hello,

the client will do one propfind to the server per folder to receive all its childs. in case of VFS, no further files are downloaded and the local items are being constructed (enumerated).
Thesw propfinds are done in parallel (batches of around 6) and also the local enumeration is being performance tuned at the moment.

So there is a fundamental difference in the initial performance of VFS and Sync.

In case of full sync, all files need to be downloaded via webdav and this can take its time.

**Comment by q-wertz** [comment]
Thanks for the clarification on the VFS part 😊
